### PR TITLE
feat: add package for TRS component

### PR DESCRIPTION
**Pull request by ltwmori**
## Description

The key changes include:

- **Setting up `ecc-client-ga4gh-trs` package:** The creation of this package under the `"packages"` folder.

- **Adding _Microsoft Fast, React, and React-scripts_:** These added dependencies are key to our project's frontend development. 

- **Creating a project structure:** This includes integrating TypeScript for type safety, and a new "components" folder within the "src" directory for housing reusable React components.

The pull request doesn't resolve any existing issues but sets up the necessary foundations for future development.

-

Fixes #(issue)

## Checklist

- [x] My code follows the [contributing guidelines][contributing] of this
      project, including, in particular, with regard to any style guidelines
- [x] The title of my PR complies with the [Conventional Commits
      specification][conventional-commits]; in particular, it clearly indicates
      that a change is a breaking change
- [x] I am aware that all my commits will be squashed into a single commit,
      using the PR title as the commit message.
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [ ] I have updated the user-facing documentation to describe any new or
      changed behavior
- [x] I have added type annotations for all function/class/method interfaces
      or updated existing ones
- [x] I have provided [JSDoc][jsdoc] block tags for all
      functions/classes/methods or updated existing ones
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature
      works
- [x] New and existing unit tests pass locally with my changes
- [x] I have not reduced the existing code coverage
- [ ] I have asked the [@all-contributors bot][all-contributors-bot] to
      acknowledge my contributions by commenting on this PR with a request of
      the form `@all-contributors please add @YOUR_GH_HANDLE for TYPE_1,
    TYPE_2, ...`, where `TYPE_1` etc. refer to [contribution types supported
      by the All Contributors Specification][all-contributors-types] OR I do
      not want my contributions to be acknowledged

## Comments
The PR does not need unit tests. It is just a set-up.

[all-contributors-bot]: https://allcontributors.org/docs/en/bot/overview
[all-contributors-types]: https://allcontributors.org/docs/en/emoji-key
[contributing]: CONTRIBUTING.md
[conventional-commits]: https://www.conventionalcommits.org/en/v1.0.0/
[jsdoc]: https://jsdoc.app/index.html

### CODE REVIEW

**Comment by git-anurag-hub** [comment]
Hey @ltwmori, If you are planning on using fast for the development. I will prefer you to use this to init your project [Official Fast CLI](https://microsoft.github.io/fast-cli/docs/initialize/).

**Comment by uniqueg** [comment]
> Hey @ltwmori, If you are planning on using fast for the development. I will prefer you to use this to init your project [Official Fast CLI](https://microsoft.github.io/fast-cli/docs/initialize/).

I think it would be good to add such comments as part of a review. Makes it easier to keep track and ensure that they have actually been resolved.

**Comment by SalihuDickson** [comment]
> Hey @ltwmori, If you are planning on using fast for the development. I will prefer you to use this to init your project [Official Fast CLI](https://microsoft.github.io/fast-cli/docs/initialize/).

Just to be clear, is this a review or just a suggestion?

**Comment by ltwmori** [comment]
> Hey @ltwmori, If you are planning on using fast for the development. I will prefer you to use this to init your project [Official Fast CLI](https://microsoft.github.io/fast-cli/docs/initialize/).

resolved!

File: packages/ecc-client-ga4gh-trs/webpack.dev.cjs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+/* eslint-disable no-undef */
+const merge = require("webpack-merge");
+const baseConfig = require("./webpack.common.cjs");
+
+module.exports = merge(baseConfig, {
+    devServer: {
+        open: true,
+        port: 3000,
+    },
+    devtool: "inline-source-map",
```
Comment (SalihuDickson) [inline packages/ecc-client-ga4gh-trs/webpack.dev.cjs:11]: There are a few options you might want to explore here, depending on how your application is performing and a few other tradeoffs. Check [here](https://webpack.js.org/configuration/devtool/) to explore your options

**Review by SalihuDickson — CHANGES_REQUESTED** [review]

File: packages/ecc-client-ga4gh-trs/src/design-system.ts
[the remark this answers is not in the document]
Comment (ltwmori) [reply packages/ecc-client-ga4gh-trs/src/design-system.ts:1]: ok! but it was generated by fast cli.

**Comment by git-anurag-hub** [comment]
@ltwmori can you please update this branch too from `main`. Otherwise, It can cause conflicts later.

**Review by SalihuDickson — APPROVED** [review]
I still think there are some things to workout here but for an initial config, this should work.

**Review by SalihuDickson — DISMISSED** [review]

**Comment by git-anurag-hub** [comment]
@ltwmori, please resolve the conflicts!

**Pull request merged**